packages/argo-checkout/src/extension-points/api/post-purchase/post-purchase.ts

```diff
@@ -1,14 +1,9 @@
 import type {StandardApi} from '../standard';
```

Comment:
Do we still need this package internally within the repo? Can we instead use the types from the `@shopify/argo-checkout` directly?

Reply:
I don't really have a strong opinion on this, I'll probably defer to you that decision since I'm still ramping up. I'm happy to remove it if we think we should.

Reply:
@denislins do you know if there was any reason that we still needed an in-repo version of the Argo API types?